R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
+            startlevel<- c(1:length(dataframe$co.names))
+            dataframe$co.names <- factor(dataframe$co.names, levels(dataframe$co.names)[startlevel])
+        }
+        
+        if (length(custom.coef.map) > 0) {
+            # keep only coefficients selected by the user and replace NAs if any
+            idx <- is.na(custom.coef.map)
+            custom.coef.map[idx] <- names(custom.coef.map)[idx]
+            selectcoef <- names(custom.coef.map)
+            keepcoef <- paste(selectcoef, collapse = "|")
+            dataframe <- dataframe[grepl(keepcoef, dataframe$co.names), ]
+            # resize number of levels in factor to avoid error
+            dataframe$co.names <- ordered(dataframe$co.names, levels = (unique(as.character(dataframe$co.names))))
+            # rename selected coefficients
+            renamedcoef <- paste(unlist(custom.coef.map), sep = ", ")
+            levels(dataframe$co.names) <- renamedcoef
+            # invert order factors for plot
+            coeforder<- c(length(dataframe$lab):1)
+            dataframe$lab <- factor(dataframe$lab, levels(dataframe$lab)[coeforder])
+        }
+        
+        # put models in the right order
+        laborder<- c(length(dataframe$lab):1)
+        dataframe$lab <- factor(dataframe$lab, levels(dataframe$lab)[laborder])
+        
+        # plot
+        p <- ggplot(dataframe, aes(lab, co)) + 
+            geom_hline(yintercept = 0, 
+                       lty = 2, 
+                       lwd = 1, 
+                       colour = "grey50") +
+            geom_errorbar(aes(ymin = lower.outer, ymax = upper.outer), 
+                          lwd = 1, 
+                          colour = ifelse(sapply(dataframe$signif, isTRUE) , signif.light, insignif.light), 
+                          width = 0) +
+            geom_errorbar(aes(ymin = lower.inner, ymax = upper.inner), 
+                          lwd = 2.5, 
+                          colour = ifelse(sapply(dataframe$signif, isTRUE), signif.medium, insignif.medium),
+                          width = 0) +
+            geom_point(size = 3, 
+                       pch = ifelse(sapply(dataframe$signif, isTRUE), 21, 22), 
+                       fill = ifelse(sapply(dataframe$signif, isTRUE), signif.dark, insignif.dark)) +
+            coord_flip() 
+        p <- p + theme
+        p <- p + xlab(" ") +
+            facet_wrap(~co.names, 
+                       strip.position = "left", 
+                       nrow = length(dataframe), 
+                       scales = "free_y")
+        
+        if (length(custom.title) > 0) {
+            p <- p + ggtitle(custom.title)
+        } 
+    } 
+    
+    # adds message to p as ylab; adds output meassages to paste that comes as R output (not in plot)
+    if (isTRUE(use.se[i]) && length(models[[i]]@se) > 0) {
+        p <- p + ylab("Bars denote SEs. Circle points denote significance.")
+        if (length(custom.note) > 0) {
+            p <- p + ylab(custom.note)
```

Comment:
Have you tested if the user can write something like `+ ylab("something")` on the R console? Can you add a unit test for this as well?

Reply:
Yes I did tried and now I added a test too